package.json

```diff
@@ -4,6 +4,9 @@
   "description": "Matrix Application Service Framework",
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "engines": {
+    "node": ">=12 18<="
```

Comment:
Are we still supporting Node 12?

Reply:
Oh, this actually doesn't work.  Testing it with https://semver.npmjs.com/ hits no matches.

What does work is any of:
- `>=x <=y`
- `x - y`

Reply:
for bonus points I've added validation of the engine string in the CI

Reply:
Ah nice, I didn't know about that feature!